util/crc32c.cc

```diff
@@ -1,7 +1,7 @@
 //  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
-//  This source code is licensed under both the GPLv2 (found in the
-//  COPYING file in the root directory) and Apache 2.0 License
-//  (found in the LICENSE.Apache file in the root directory).
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
```

Comment:
why change the license? i think only the copyright holder(s) is allowed to do so.

Reply:
I didn't really changed it, but only kept the same as what we have on the rocksdb commit used by Ceph, which is not the rocksdb:master by the way.